File: tools/src/main/java/com/yinp/tools/fragment_dialog/ViewConvertListener.java
package com.yinp.tools.fragment_dialog;

import androidx.viewbinding.ViewBinding;

/**
 * 用于dialogFragment和Activity的通讯
 */
public abstract class ViewConvertListener {
   public void convertView(DialogFragmentHolder holder, BaseDialogFragment dialogFragment) {
    }
    public void convertView(DialogFragmentHolder holder, BaseDialogFragment dialogFragment, ViewBinding viewBinding) {
    }
}

File: tools/src/main/java/com/yinp/tools/fragment_dialog/ViewConvertListener.kt
package com.yinp.tools.fragment_dialog

import androidx.viewbinding.ViewBinding

/**
 * author:yinp
 * 2022/4/25+20:38
 * describe:
 */
abstract class ViewConvertListener {
    open fun convertView(holder: DialogFragmentHolder, dialogFragment: BaseDialogFragment) {}
    open fun convertView(
        holder: DialogFragmentHolder,
        dialogFragment: BaseDialogFragment,
        viewBinding: ViewBinding
    ) {
    }
}